SimPEG/electromagnetics/base_1d.py

```diff
@@ -299,19 +366,25 @@ def compute_complex_mu(self, frequencies):
         mu = np.tile(mu.reshape([-1, 1]), (1, n_frequency))
 
         # No magnetic viscosity
-        if np.all(self.dchi) == 0.0:
+        if np.all(self.cole_cole_parameters.chi) == 0.0:
             return mu
 
         # Magnetic viscosity
         else:
-            if np.isscalar(self.dchi):
-                dchi = self.dchi * np.ones_like(self.mu)
-                tau1 = self.tau1 * np.ones_like(self.mu)
-                tau2 = self.tau2 * np.ones_like(self.mu)
+            if np.isscalar(self.cole_cole_parameters.chi):
+                dchi = self.cole_cole_parameters.chi * np.ones_like(self.mu)
+                tau1 = self.cole_cole_parameters.tau1 * np.ones_like(self.mu)
+                tau2 = self.cole_cole_parameters.tau2 * np.ones_like(self.mu)
             else:
-                dchi = np.tile(self.dchi.reshape([-1, 1]), (1, n_frequency))
-                tau1 = np.tile(self.tau1.reshape([-1, 1]), (1, n_frequency))
-                tau2 = np.tile(self.tau2.reshape([-1, 1]), (1, n_frequency))
+                dchi = np.tile(
+                    self.cole_cole_parameters.chi.reshape([-1, 1]), (1, n_frequency)
+                )
+                tau1 = np.tile(
+                    self.cole_cole_parameters.tau1.reshape([-1, 1]), (1, n_frequency)
+                )
+                tau2 = np.tile(
+                    self.cole_cole_parameters.tau2.reshape([-1, 1]), (1, n_frequency)
```

Comment:
Not sure we actually need to tile the values. Can't we just pass the same vector to all frequencies?